planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -137,3 +159,54 @@ function validatePlaceDuration(duration) {
     }
     return true;
 }
+
+async function submitPlaceCallback(place, status) {
+    if (status == google.maps.places.PlacesServiceStatus.OK) {
```

Comment:
Consider to have some kind of error message to let users know their submission failed.

Reply:
Fixed